.github/workflows/rust.yml

```diff
@@ -144,6 +145,11 @@ jobs:
           echo "--- Forest STDOUT ---"; cat forest.out
           echo "--- Forest STDERR ---"; cat forest.err
           echo "--- Forest Prometheus metrics ---"; cat metrics.log
+      - name: export snapshot and sync forest with exported snapshot
+        run: |
+          rm $SNAPSHOT_DIRECTORY/*
+          forest-cli snapshot export -o $SNAPSHOT_DIRECTORY
```

Comment:
You can't export a snapshot without having a running node.

Reply:
`forest-cli` does pretty much nothing on its own. It just sends messages to the Forest node.

Reply:
ah i reordered this step incorrectly. fixed.